README.md

```diff
@@ -27,13 +32,17 @@ To use `mksurfdata.pl`, regridding weights have to be created. For this the grid
 - ESMF-Mesh 
 - CF-convention-file
 
-SCRIP is a very old format not maintained anymore but is still the most effective solution. it can be easily created with curvilinear_to_SCRIP from NCL. Unfortunately, NCL is also not maintained anymore, so this could lead to problems in the future.
+SCRIP is a very old format not maintained anymore but is still the most effective solution. it can be easily created with `curvilinear_to_SCRIP` from NCL. Unfortunately, NCL is also not maintained anymore, so this could lead to problems in the future.
 
-ESMF-Mesh files are able to describe unstructured grids. A solution to create them is again found in a CTSM branch: `subset_mesh_dask` under `tools/site_and_regional/mesh_maker.py` I had to rewrite part of the script for it to accept 2d lat / lon. The main caveat is that the resulting surface files are in 1D which makes them harder to handle.
+ESMF-Mesh files are able to describe unstructured grids.
+You can create the gridfile with [a script](mkmapgrids/scrip_mesh.py).
+You need to install the Python packages numpy, xarray and dask-expr for this.
```

Comment:
Suggestion instead of `You need to install the Python packages numpy, xarray and dask-expr for this.`: The Python packages numpy, xarray and dask-expr needs to be available.

One remark: I have used `ICON_scrip.py` to generate the grid files. More discussion can be done in issue #8